src/common/cvar.c

```diff
@@ -89,7 +89,11 @@ replacement_t replacements[] = {
 	{"gl_shadows", "r_shadows"},
 	{"gl_anisotropic", "r_anisotropic"},
 	{"gl_lightmap", "r_lighmap"},
-	{"intensity", "gl1_intensity"}
+	{"intensity", "gl1_intensity"},
```

Comment:
Intensity -> r_intensity
Gl1_intensity -> r_intensity
Vk_intensity -> r_intensity

As know, name of variable should be also changed in {gl1,gl3}_main.c, e.g. https://github.com/yquake2/yquake2/blob/master/src/client/refresh/gl1/gl1_main.c#L1230 

Vk_ref change could be separate PR, if/when current PR is approved.

And check of current render will be not required as all variables will be same.